Services/ItemService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Services.DTO;
+using System;
+using System.Collections.Generic;
+
+namespace Services
+{
+    public class ItemService
+    {
+        private static readonly string[] Items = new[]
+        {
+            "One", "Two", "Three"
+        };
+
+        public IEnumerable<string> GetAll(int userId)
+        {
+            // no requirements on what to do on the userId
+            return Items;
+        }
+
+
+        public void Save(ItemDTO itemObject)
+        {
+            Console.Write(itemObject);
```

Comment:
Please map ItemDTO to an Item domain model here. You can use your constructor approach just create an instance of Item object

Reply:
this is updated